packages/blade/src/components/Box/Box.tsx

```diff
@@ -147,6 +151,16 @@ const makeBoxProps = (props: BoxProps): KeysRequired<Omit<BoxProps, 'testID' | '
     onMouseOver: props.onMouseOver,
     onScroll: props.onScroll,
 
+    // Drag and Drop
+    draggable: props.draggable,
+    onDragStart: props.onDragStart,
+    onDragEnd: props.onDragEnd,
+    onDragEnter: props.onDragEnter,
+    onDragLeave: props.onDragLeave,
+    onDragOver: props.onDragOver,
+    onDrop: props.onDrop,
```

Comment:
I think the event handlers and `draggable` will get handled by themselves (since they are HTML attributes and not CSS properties) but you might have to add pointerEvents, placeItems, and other props to baseBoxStyles